### ISSUE TITLE
NoClassDefFoundError on server start-up

### ISSUE THREAD

**Issue by SlimeDog** [Zrips/Residence#686]
Spigot 1.16.5-b3024
Residence 4.9.3.0

On server start-up:
```
[09:44:40] [Server thread/ERROR]: Could not load 'plugins/Residence-4.9.3.0.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: java.lang.NoClassDefFoundError: com/griefcraft/scripting/Module
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:139) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:394) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:301) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.loadPlugins(CraftServer.java:382) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:224) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:918) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:273) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NoClassDefFoundError: com/griefcraft/scripting/Module
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:398) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:64) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:135) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        ... 7 more
Caused by: java.lang.ClassNotFoundException: com.griefcraft.scripting.Module
        at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:167) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:96) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:398) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:64) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:135) ~[spigot-1.16.5-3024.jar:3024-Spigot-79d53c2-90d6905]
        ... 7 more
```

**Comment by mercurialmusic** [comment]
Same. https://gist.github.com/mercurialmusic/92ff2a8d85dd31f386b6eb90108ec873

**Comment by SlimeDog** [comment]
So something changed in Spigot internals. Paper just caught up.

**Comment by Zrips** [comment]
9d0786c95f2e58c797ca5789ef577596fd4d5bef
Posted new version.

**Comment by 100petr** [comment]
Still @Zrips https://pastebin.com/ZidLJJM9

**Comment by SlimeDog** [comment]
Confirmed fixed on Spigot with 4.9.3.2

**Comment by 100petr** [comment]
Update #688

**Comment by SlimeDog** [comment]
Confirmed fixed on Paper with 4.9.3.2
Thanks.
